js/client-mainmenu.js

```diff
@@ -295,6 +296,7 @@
 							$chat.append('<div class="chat">User ' + userid + ' no longer ignored.</div>');
 						}
 					} else {
+						this.chatHistories[userid].push(text);
```

Comment:
Why is this here? Shouldn't `/ignore` and `/unignore` be added to the histories?